frontend/android/fastlane/Fastfile

```diff
@@ -144,11 +144,11 @@ platform :android do
 
     build_config_name = options[:build_config_name]
 
-    if [version_name, version_code, production_delivery, build_config_name].include?(nil)
+    if [build_config_name].include?(nil)
       raise "'nil' passed as parameter! Aborting..."
     end
 
-    application_id = sh("app-toolbelt v0 build-config to-json #{build_config_name} android | jq .applicationId")
+    application_id = sh("app-toolbelt v0 build-config to-json #{build_config_name} android | jq -rj .applicationId ")
```

Comment:
what does this `-rj` stand for? empty space after `applicationId` can be removed i think

Reply:
-r `output raw strings, not JSON texts`
-j `eliminate literal newlines`